CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ catkin_package()
 
 roslaunch_add_file_check(launch)
 
-install(DIRECTORY config launch
+catkin_install_python(PROGRAMS scripts/stow_ur5.py DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```

Comment:
Best to leave off the `.py` extension; gives the freedom to change the impl later, and a nicely command-line interface, eg:

    rosrun husky_control stow_ur5